apps/condo/domains/subscription/hooks/useSubscriptionPaymentModal.tsx

```diff
@@ -14,31 +12,33 @@ interface IApplySubscriptionModal {
 
 export const useSubscriptionPaymentModal = (): IApplySubscriptionModal => {
     const intl = useIntl()
+    const ModalTitle = intl.formatMessage({ id: 'subscription.modal.complete.title' })
```

Comment:
Same here....seems like all `useBlahBlahModal` are redundant, if you're spawning it from user, if not - than it' viable option